src/libs/actions/User.js

```diff
@@ -69,7 +69,7 @@ function getUserDetails() {
             // Update the User onyx key
             const loginList = _.where(response.loginList, {partnerName: 'expensify.com'});
             const expensifyNewsStatus = lodashGet(response, 'account.subscribed', true);
-            Onyx.merge(ONYXKEYS.USER, {loginList, expensifyNewsStatus});
+            Onyx.merge(ONYXKEYS.USER, {loginList, expensifyNewsStatus: !!expensifyNewsStatus});
```

Comment:
Curious why we're using evaluating to a truthy value here, what is the expected value of `expensifyNewsStatus`? A number?

Reply:
the Api returns Int 1, 0 and we want to save Boolean

Reply:
makes sense, thanks!